src/nvim/ops.c

```diff
@@ -5310,6 +5312,12 @@ static void free_register(struct yankreg *reg)
   y_current = curr;
 }
 
+int get_unnamed_register_name(void) {
+    int name = 0;
+    adjust_clipboard_name(&name, true, false);
```

Comment:
:+1: Right now I have a leader mapping to set all shifts to 2, but I should add an autocommand or something for all files under `~/dev/neovim`